rf-stylez.gemspec

```diff
@@ -20,6 +20,7 @@ Gem::Specification.new do |spec|
   spec.require_paths = ['lib']
 
   spec.add_runtime_dependency 'rubocop', '>= 0.59', '< 0.81'
+  spec.add_runtime_dependency 'rubocop-rails', '~> 2.5', '>= 2.5.0'
```

Comment:
What do you think about locking the minor as well, cause, a quick look at the changelog reveals that they add new cops to new minors so different devs will have different sets of cops depending on when they ran `gem install rf-stylez`
```suggestion
  spec.add_runtime_dependency 'rubocop-rails', '~> 2.5.0'
```
Also `>=` is not needed in such case

Reply:
Sounds good to me, I was just using the suggestion that showed up when I built it locally.